src/modules/quickphrase/gen_emoji_eac.py

```
@@ -0,0 +1,31 @@
+import os
+
+import requests
+
+EMOJI_JSON_URL = "https://raw.githubusercontent.com/delthas/gemoji-json/master/emoji.json"
+OUTPUT = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+    "src/modules/quickphrase/quickphrase.d/emoji-eac.mb",
+)
+
+print(f"Downloading {EMOJI_JSON_URL}")
+response = requests.get(EMOJI_JSON_URL)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)